controllers/datafoundation.go

```diff
@@ -114,3 +146,68 @@ func (r *dataFoundationReconciler) reconcileOnboardingValidationSecret() error {
 	}
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileCSV() error {
+	r.Log.Info("Reconciling OCS Operator CSV")
+	csvList := opv1a1.ClusterServiceVersionList{}
+	if err := r.list(&csvList); err != nil {
+		return fmt.Errorf("unable to list csv resources: %v", err)
+	}
+
+	for index := range csvList.Items {
+		csv := &csvList.Items[index]
+		if strings.HasPrefix(csv.Name, ocsOperatorName) {
+			if err := r.updateOCSCSV(csv); err != nil {
+				return fmt.Errorf("failed to update OCS CSV: %v", err)
+			}
+			break
+		}
+	}
+	return nil
+}
+
+func (r *dataFoundationReconciler) updateOCSCSV(csv *opv1a1.ClusterServiceVersion) error {
+	isChanged := false
+	if _, found := csv.Annotations[managedFusionAnnotation]; !found {
+		utils.AddAnnotation(
+			csv,
+			managedFusionAnnotation,
```

Comment:
This is not local to the offering so it makes no sense to have access to it here. Maybe you want an operation like `r.addOwnerAnnotation(res client.Object)` added to the embedded reconciler